assets/onboarding/onboarding.jsx

```diff
@@ -0,0 +1,19 @@
+import { __ } from '@wordpress/i18n';
+
+import { Button } from '@wordpress/components';
+import { Card } from '@woocommerce/components';
+import '@woocommerce/components/build-style/style.css';
+
+function SenseiOnboardingPage() {
+	return (
+		<Card className="wrap">
+			<h1> { __( 'Welcome to Sensei LMS!', 'sensei-lms' ) } </h1>
+			<Button isPrimary>Okay</Button>
+		</Card>
+	);
+}
+
+wp.element.render(
```

Comment:
To keep the consistency, we could import the render from the element package. WDYT? `import { render } from '@wordpress/element';`

Reply:
Good idea, changed it to this.